src/services/__tests__/classService.simple.test.ts

```
@@ -0,0 +1,254 @@
+                    name: 'Class A',
+                    class_teachers: [{ user_id: 'teacher-123', role: 'primary', profiles: { full_name: 'Teacher Name' } }],
+                    class_students: [],
+                    class_subjects: []
+                }
+            ];
+
+            const mockQuery = {
+                select: jest.fn().mockReturnThis(),
+                eq: jest.fn().mockReturnThis(),
+                order: jest.fn().mockReturnThis(),
+                then: jest.fn().mockResolvedValue({
```

Biome (2.1.2)

[error] Do not add then to an object.



(lint/suspicious/noThenProperty)